docs/Tutorial_DiscordMERLIN.ipynb

```diff
@@ -0,0 +1,1356 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #41.            if not np.any(is_cands):
</pre><p>This feels hard to read and understand. It is common that if you have to leave a comment into the code like <code>means all candidates are pruned! no need to check further! There is nothing to return!</code> then it suggests that the code itself is not clear. Maybe something like <code>if np.all(is_cands == False):</code>  is clearer in expressing that <code>is_cands</code>  is all <code>False</code> . It might be slightly slower but I think it's worth it.</p><p> </p><p>Same thing about:</p><p><code>if not len(non_trivial_cand):</code></p><p> </p><p>Btw, should it be <code>non_trivial_cands</code> (plural)?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
> This feels hard to read and understand.

Agreed. Will be fixed.

> Btw, should it be non_trivial_cands (plural)?
If you mean we might have more than one candidate here, you are right. I will change the name. Thanks for catching that!

Reply:
<p>Yes, that's what I mean. Thank you</p>